.gitignore

```diff
@@ -29,6 +29,7 @@ bld/
 temp/
 _common/
 _build/
+_download/
```

Comment:
it was using _temp before to download externals. i can switch it back, it just felt like _download was more clean. i'm on the fence and fine either way